### PR TITLE
🚀[Release v3.10.1] Merge into Develop

### PR DESCRIPTION
**Pull request by kcw-grunt**
# Overview
Fixed some code issues

## What's Changed
Release/v3.9.0 by @kcw-grunt in https://github.com/litecoin-foundation/litewallet-ios/pull/128
🚀[Release v3.10.0] Merge into Develop by @kcw-grunt in https://github.com/litecoin-foundation/litewallet-ios/pull/135
🚧[Techdebt] Firebase polish 121 by @kcw-grunt in https://github.com/litecoin-foundation/litewallet-ios/pull/139
Mark unused code by @Iferencak in https://github.com/litecoin-foundation/litewallet-ios/pull/147
🦟 [Bugfix] buy fixes 138 by @kcw-grunt in https://github.com/litecoin-foundation/litewallet-ios/pull/150
Retired old https server code by @Iferencak in https://github.com/litecoin-foundation/litewallet-ios/pull/154
https://github.com/litecoin-foundation/litewallet-ios/issues/148 TransactionsViewController crashfix by @Iferencak in https://github.com/litecoin-foundation/litewallet-ios/pull/155
## New Contributors
@Iferencak made their first contribution in https://github.com/litecoin-foundation/litewallet-ios/pull/147

### CODE REVIEW

File: loafwallet.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3985,25 +3984,6 @@
...
-			);
-			name = "Check for unused code";
-			outputFileListPaths = (
-			);
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "file=\"unused.rb\"\nif [ -f \"$file\" ]\nthen\necho \"$file found.\"\nruby unused.rb xcode\nelse\necho \"unused.rb doesn't exist\"\nfi\n";
-		};
```
Comment (Iferencak) [inline loafwallet.xcodeproj/project.pbxproj:4006]: @kcw-grunt should this be removed?

Comment (Iferencak) [reply loafwallet.xcodeproj/project.pbxproj:4006]: Oh, I see this was a cleanup of duplicate scripts. If this was a duplicate than it's ok.

**Review by Iferencak — APPROVED** [review]

File: loafwallet.xcodeproj/project.pbxproj (hunk 1 above)
Comment (kcw-grunt) [reply loafwallet.xcodeproj/project.pbxproj:4006]: yup...it was a dupe.  When I was resolving conflicts @Iferencak

**Comment by kcw-grunt** [comment]
note I also bumped the version to test distribution @Iferencak ....this is not normal 
but AS rejects because they already approved v3.10.1

File: loafwallet/src/Strings/it.lproj/Localizable.strings [hunk 2]
```diff
@@ -974,7 +974,7 @@
 "Send.UnstoppableDomains.simpleplaceholder" = "Inserisci dominio";
 
 /* UDSystemError */
-"Send.UnstoppableDomains.udSystemError" = "Problema di ricerca nel sistema. [Errore: %2$d]";
+"Send.UnstoppableDomains.udSystemError" = "Sistem arama sorunu. [Hata:%2$d]";
```
Comment (Iferencak) [inline loafwallet/src/Strings/it.lproj/Localizable.strings]: Sorry, I know this is mega-minor, but just to keep it consistent, could we add one space after Hata:?
`"Send.UnstoppableDomains.udSystemError" = "Sistem arama sorunu. [Hata: %2$d]";`

Comment (kcw-grunt) [reply loafwallet/src/Strings/it.lproj/Localizable.strings]: Oh hell yeah.  Thanks @Iferencak I totally agree.

**Pull request merged**